spinalcordtoolbox/scripts/sct_deepseg.py

```diff
@@ -149,9 +152,18 @@ def main(argv=None):
     if arguments.task is None:
         parser.error("You need to specify a task.")
 
-    # Check if all input images are provided
-    required_contrasts = deepseg.models.get_required_contrasts(arguments.task)
-    if len(arguments.i) != len(required_contrasts):
+    # Verify if the task is custom
```

Comment:
the primary clause of the if is to check if task is official (the "custom" is the else)
```suggestion
    # Verify if the task is part of the "official" tasks, or if it is pointing to paths containing custom models
```